### ISSUE TITLE
Autofill not working on latest stable (2.10.0)

### ISSUE THREAD

**Issue by mkranus** [flutter/flutter#98112]
Autofill is not working when using 2.10.0 (also didn't work for 2.8.x). This problem was also reported in another issue, but its still not solved although the issue was closed.

older issue: https://github.com/flutter/flutter/issues/94043

**Comment by maheshmnj** [comment]
Hi @mkranus, Thanks for filing the issue. Could you please share what target platform is not working?

**Comment by mkranus** [comment]
@maheshmnj its not working on Android and iOS

**Comment by maheshmnj** [comment]
Thanks for the info, AutoFill for Android is fixed on latest master see this discussion https://github.com/flutter/flutter/issues/97633

Ios specific issue is being tracked here https://github.com/flutter/flutter/issues/91354

Closing as the duplicate of the above linked issues.

**Comment by maheshmnj** [comment]
@mkranus,  AutoFill works on the master channel and is planned for a hotfix on stable 2.10.0 Please see this https://github.com/flutter/flutter/issues/97633#issuecomment-1033949522. I believe your issue should be fixed by this release.

**Comment by mkranus** [comment]
@maheshmnj The iOS issue you attached earlier is different https://github.com/flutter/flutter/issues/91354. My problem is that it is completely not working.

**Comment by maheshmnj** [comment]
@mkranus I do see that it has landed on `stable 2.10.1` https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel. I will try to reproduce and update you on this.

**Comment by maheshmnj** [comment]
@mkranus sorry for the delay I can confirm that autofill for Android works on `stable 2.10.2`, I verified the above code sample.

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 2.10.2, on macOS 12.1 21C52 darwin-arm, locale en-GB)
    • Flutter version 2.10.2 at /Users/mahesh/Documents/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 097d3313d8 (2 days ago), 2022-02-18 19:33:08 -0600
    • Engine revision a83ed0e5e3
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.10.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Community Edition (version 2021.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 61.2.4
    • Dart plugin version 212.5080.8

[✓] VS Code (version 1.63.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.34.0

[✓] Connected device (3 available)
    • Redmi K20 Pro (mobile) • 192.168.1.2:5555 • android-arm64  • Android 11 (API 30)
    • macOS (desktop)        • macos            • darwin-arm64   • macOS 12.1 21C52 darwin-arm
    • Chrome (web)           • chrome           • web-javascript • Google Chrome 98.0.4758.102

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>
